test/integration/default/serverspec/server_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'serverspec'
+
+set :backend, :exec
+
+describe service('varnishlog') do
```

Comment:
Its probably a good idea to add a few simple tests for varnish itself. Make sure the service is running with the correct port, etc. Also check the admin port is there. Do we want to include the firewall recipe for http here too? I think there's a varnish recipe too?